tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -355,12 +355,22 @@ def test_data_based_criterion(mode, ref_scores, ref_act_scores, mocker):
 
 
 @pytest.mark.parametrize("mode", (CompressWeightsMode.INT8_SYM, CompressWeightsMode.INT8_ASYM))
-def test_not_quantize_with_multiple_reduction_axes(mode):
+def test_quantize_Gather_with_multiple_reduction_axes_in_8bit(mode):
```

Comment:
Please, add the same test for the torch backend.

Reply:
According to https://github.com/openvinotoolkit/nncf/blob/2bde5957c318997c8c81ad462deb39751a85ef9d/nncf/quantization/algorithms/weight_compression/torch_backend.py#L131-L132 `torch.nn.Embedding` can't have multiple reduction axes.